servant-server/src/Servant/Server/Internal.hs

```diff
@@ -22,43 +22,40 @@ module Servant.Server.Internal
-                                             httpVersion, isSecure,
-                                             lazyRequestBody,
-                                             rawQueryString, remoteHost,
-                                             requestHeaders, requestMethod,
-                                             responseLBS, vault)
-import           Prelude                    ()
+import           GHC.TypeLits                (KnownNat, KnownSymbol, natVal,
+                                              symbolVal)
+import           Network.HTTP.Types          hiding (Header, ResponseHeaders)
+import           Network.Socket              (SockAddr)
+import           Network.Wai                 (Application, Request, Response,
+                                              httpVersion, isSecure,
+                                              lazyRequestBody, rawQueryString,
+                                              remoteHost, requestHeaders,
+                                              requestMethod, responseLBS, vault)
+import           Prelude                     ()
 import           Prelude.Compat
-import           Web.HttpApiData            (FromHttpApiData)
-import           Web.HttpApiData.Internal   (parseHeaderMaybe,
-                                             parseQueryParamMaybe,
-                                             parseUrlPieceMaybe,
-                                             parseUrlPieces)
-
-import           Servant.API                 ((:<|>) (..), (:>), BasicAuth, Capture,
-                                              CaptureAll, Verb,
-                                              ReflectMethod(reflectMethod),
-                                              IsSecure(..), Header, QueryFlag,
+import           Web.HttpApiData             (FromHttpApiData, parseHeaderMaybe,
+                                              parseQueryParamMaybe,
+                                              parseUrlPieceMaybe,
+                                              parseUrlPieces)
+
+import           Servant.API                 ((:<|>) (..), (:>), BasicAuth,
+                                              Capture, CaptureAll, Header,
+                                              IsSecure (..), QueryFlag,
                                               QueryParam, QueryParams, Raw,
-                                              RemoteHost, ReqBody, Vault,
+                                              ReflectMethod (reflectMethod),
+                                              RemoteHost, ReqBody, Vault, Verb,
                                               WithNamedContext)
 import           Servant.API.ContentTypes    (AcceptHeader (..),
                                               AllCTRender (..),
-                                              AllCTUnrender (..),
-                                              AllMime,
+                                              AllCTUnrender (..), AllMime,
```

Comment:
Well, personally I try very hard not to make formatting changes in normal PRs. Just to keep the noise level down and to avoid (possibly tool-supported) edit wars. Otherwise I would also change formatting all the time. So from my perspective: yes, please revert.